src/pages/stablecoins.tsx

```diff
@@ -11,19 +11,12 @@ import Emoji from "../components/OldEmoji"
 import FeedbackCard from "../components/FeedbackCard"
 import GhostCard from "../components/GhostCard"
 import HorizontalCard from "../components/HorizontalCard"
-import Icon from "../components/Icon"
 import InfoBanner from "../components/InfoBanner"
 import Link from "../components/Link"
 import PageHero from "../components/PageHero"
 import PageMetadata from "../components/PageMetadata"
 import ProductList from "../components/ProductList"
-import {
-  CardGrid,
-  Content,
-  Divider,
-  GradientContainer,
-  Page,
-} from "../components/SharedStyledComponents"
+import { Content, Divider, Page } from "../components/SharedStyledComponents"
```

Comment:
We are deprecating the `SharedStyledComponents` library so we also need to refactor the places where we are using them and use a Chakra component instead.